WalletWasabi/WabiSabi/Backend/Rounds/Arena.Partial.cs

```diff
@@ -38,6 +38,11 @@ private async Task<InputRegistrationResponse> RegisterInputCoreAsync(InputRegist
 		{
 			var round = GetRound(request.RoundId);
 
+			if (round.AlreadyRegisteredInputs.Contains(coin.Outpoint))
+			{
+				throw new WabiSabiProtocolException(WabiSabiProtocolErrorCode.AliceAlreadyRegistered);
+			}
+
```

Comment:
Isn't this check too similar to the check below (L49-L51) where we throw the same `ProtocolException`?